frontend/app/.server/domain/services/federal-government-insurance-plan.service.ts

```diff
@@ -3,15 +3,49 @@ import moize from 'moize';
 
 import type { ServerConfig } from '~/.server/configs';
 import { SERVICE_IDENTIFIER } from '~/.server/constants';
-import type { FederalGovernmentInsurancePlanDto } from '~/.server/domain/dtos';
+import type { FederalGovernmentInsurancePlanDto, FederalGovernmentInsurancePlanLocalizedDto } from '~/.server/domain/dtos';
 import { FederalGovernmentInsurancePlanNotFoundException } from '~/.server/domain/exceptions';
 import type { FederalGovernmentInsurancePlanDtoMapper } from '~/.server/domain/mappers';
 import type { FederalGovernmentInsurancePlanRepository } from '~/.server/domain/repositories';
 import type { LogFactory, Logger } from '~/.server/factories';
 
+/**
+ * Service interface for managing federal government insurance plan data.
+ */
 export interface FederalGovernmentInsurancePlanService {
-  listFederalGovernmentInsurancePlans(): FederalGovernmentInsurancePlanDto[];
+  /**
+   * Retrieves a list of all federal government insurance plans.
+   *
+   * @returns An array of FederalGovernmentInsurancePlan DTOs.
+   */
   getFederalGovernmentInsurancePlanById(id: string): FederalGovernmentInsurancePlanDto;
+
+  /**
+   * Retrieves a specific federal government insurance plan by its ID.
+   *
+   * @param id - The ID of the federal government insurance plan to retrieve.
+   * @returns The FederalGovernmentInsurancePlan DTO corresponding to the specified ID.
+   * @throws {FederalGovernmentInsurancePlanNotFoundException} If no federal government insurance plan is found with the specified ID.
+   */
+  listFederalGovernmentInsurancePlans(): ReadonlyArray<FederalGovernmentInsurancePlanDto>;
```

Comment:
The documentation for these two methods should be flipped.

Reply:
Fixed